### ISSUE TITLE
best checkpoint

### ISSUE THREAD

**Issue by BetterZH** [fawazsammani/nlxgpt#12]
Thank you for the response to question[ 1](https://github.com/fawazsammani/nlxgpt/issues/11#issue-1776035898). Do you mean 2-3h or 6h is the time for training 30 epochs? 
And how to determine which epoch achieves the optimal model? Is it the epoch where a certain metric (such as B-4) achieves the best value on the validation set? Or something else?

**Comment by BetterZH** [comment]
In addition, I use a single RXT 2080ti GPU to finetune the model on vqaX dataset by `accelerate launch vqaX.py`. The utilization rate of GPU is 20%, and the training time of one epoch is about 45 minutes. I don't know why my training speed is so slow. Is the GPU utilization rate normal?

**Comment by fawazsammani** [comment]
Yes. We took the epoch which has the best metric result (usually all of them are correlated) on the validation split. 

VQA-X takes roughly 2-3 hours for 30 epochs
e-SNLI-VE takes roughly 6 hours

But this depends on your GPU. We were using 2080Ti that time.
